### ISSUE TITLE
authority of variety

### ISSUE THREAD

**Issue by sjmeades** [Canadensys/vascan-data#1954]
[Originally posted on GoogleCode (id 2087) on 2014-01-30]

<b>(This is the template to report a data issue for Vascan. If you want to</b>
<b>report another issue, please change the template above.)</b>

<b>What is the URL of the page where the problem occurs?</b>
http://data.canadensys.net/vascan/taxon/8122

<b>What data are incorrect or missing?</b>
Eriogonum strictum subsp. proliferum (Torrey &amp; A. Gray) S. Stokes var. proliferum

<b>What data are you expecting instead?</b>
Eriogonum strictum subsp. proliferum var. proliferum (Torrey &amp; A. Gray) C.L.Hitchcock

<b>If applicable, please provide an authoritative source.</b>
IPNI, MBG

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2014-01-31 19:02Z]

The authorities were indeed misapplied here. Tx

there is a further issue: the program appears to see this as a case similar to autonyms (ie, authorities only after first name), but it is not!

To go around this, I simplified the taxonomy to that of FNA, ie, only var. - the tax. used was that of Hitchcock (subsp. with var.)

Luc
